shared/bus.go

```diff
@@ -21,6 +23,8 @@ type bus struct {
 	utility     modules.UtilityModule
 	consensus   modules.ConsensusModule
 	telemetry   modules.TelemetryModule
+
+	clock clock.Clock
```

Comment:
I really like the ability and design of injecting a clock into the node and bus. 

Any chance you could add some detail (short & concise + diagram additions/updates) to `shared/README.md`?

Reply:
Maybe addressed in https://github.com/pokt-network/pocket/pull/198/commits/e1d579283f0476e87a17708dfc9963f3e48bf345

Reply:
LGTM